.github/workflows/tests.yml

```diff
@@ -78,7 +71,7 @@ jobs:
         run: python -m pip install -i https://test.pypi.org/simple/ PennyLane-Lightning --pre --upgrade
 
       - name: Run tests
-        run: python -m pytest tests --cov=pennylane $COVERAGE_FLAGS
+        run: python -m pytest tests --cov=pennylane $COVERAGE_FLAGS -m "not qcut and not qchem" -n auto

```

Comment:
`-n` does the distribution using `pytest-xdist` and `auto` will attempt to use as many cores as there are available. It could be `-n 2` too for example.